.env

```diff
@@ -0,0 +1,10 @@
+SECRET_KEY=@e2(yx)v&tgh3_s=0yja-i!dpebxsz^dg47x)-k&kq_3zf*9e*
+DEBUG=True
+ALLOWED_HOSTS=localhost,127.0.0.1
+DATABASE_ENGINE=django.db.backends.sqlite3
+DATABASE_NAME=db.sqlite3
+LANGUAGE_CODE=en-us
+TIME_ZONE=UTC
+SECRET_KEY=ecret_key_here
+DEBUG=True
+ALLOWED_HOSTS=localhost,127.0.0.1
```

Comment:
This file should be in gitignorе because it contains important data.
<img width="628" alt="image" src="https://github.com/user-attachments/assets/3a60918e-34f4-49a1-8935-bf4d78ce12aa" />